### PR TITLE
RA-1693 : Display right message and show succes message on deleting form data in encounter view mode

### PR DESCRIPTION
**Pull request by mozzy11**
Ticket : https://issues.openmrs.org/browse/RA-1693

Display a succes message on deleting a form in Encounter view mode

### CODE REVIEW

File: api/src/main/resources/messages.properties [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ${project.parent.artifactId}.title=HTML Form Entry UI Framework Integration Module
 
 htmlformentryui.viewHtmlForm.breadcrumb = {0}
-htmlformentryui.thisFormIsDeleted = This form has been deleted
+htmlformentryui.thisFormIsDeleted = This form's data has been deleted
```
Comment (dkayiwa) [inline api/src/main/resources/messages.properties]: In your screenshot, i do not see any place where the above message is displayed.

**Comment by mozzy11** [comment]
hi @dkayiwa , i attached the screenhot there on the ticket

File: api/src/main/resources/messages.properties (hunk 1 above)
Comment (mozzy11) [reply api/src/main/resources/messages.properties]: i attached thescreenshot there

**Comment by mozzy11** [comment]
@dkayiwa  , can you please help merge this   ??

**Comment by dkayiwa** [comment]
Take another look at the ticket description.

**Comment by mozzy11** [comment]
i updated it again

**Comment by dkayiwa** [comment]
Finally you get it right. 😊

**Comment by mozzy11** [comment]
hahah ,hmmm. i like the way you deal with issues

**Pull request merged**